packages/integration-objects/srv/integration-objects/simple-exchange-rate-type-service.js

```diff
@@ -0,0 +1,210 @@
+/* Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. */
+const cds = require('@sap/cds');
+const util = require('@sap-cloud-sdk/util');
+const Constants = require('../utils/Constants');
```

Comment:
This file does not exist on this branch - why aren't the tests failing? 🤔

Reply:
It's in the main. I have merged with main now.

Reply:
👍